modules/tinymce/src/plugins/image/test/ts/browser/plugin/MainTabTest.ts

```diff
@@ -47,4 +48,29 @@ describe('browser.tinymce.plugins.image.plugin.MainTabTest', () => {
       '</figure>'
     ));
   });
+
+  it('TINY-6400: render image_class_list as Dialog type "panel"', async () => {
+    const editor = hook.editor();
+    editor.settings.image_caption = false;
```

Comment:
This needs to be cleaned up at the end of the test so it doesn't impact other tests in this file. eg:
```
editor.settings.image_caption = true;
```

- [ ] Clean up the setting that was changed in this test case